packages/data-point/src/ReducerPath.js

```diff
@@ -39,21 +97,54 @@ function getPath(value, path) {
   return current;
 }
 
-class ReducerPath extends ReducerNative {
+/**
+ * @param {Accumulator} accumulator
+ * @param {Object} compiledPath
+ * @returns {any} resolved value
+ */
+function resolvePath(accumulator, compiledPath) {
+  const valueToCrawl =
+    compiledPath.moveUp === true ? accumulator : accumulator.value;
```

Comment:
moveUp could have a more specific name. something like `useAcculatorAsSource`?

...or, instead of having moveUp, the compiledPath array could (by default) have "value" prepended to it. then, this function wouldn't need to know if it's "moving up" or not. it would just access a property on the accumulator, and that path may or may not begin with "value"

Reply:
I tinkered with all of the solutions and the simplest is the renaming, so fixed with that solution